llm/vllm/serve.yaml

```diff
@@ -0,0 +1,33 @@
+resources:
+  accelerators: A100-80GB:8
+
+setup: |
+  conda activate vllm
+  if [ $? -ne 0 ]; then
+    conda create -n vllm python=3.9 -y
+    conda activate vllm
+  fi
+
+  git clone https://github.com/vllm-project/vllm.git || true
+
+  cd vllm
+  pip list | grep vllm || pip install .
+  pip install gradio
+
+
+run: |
+  conda activate vllm
+  echo 'Starting vllm api server...'
+  python -u -m vllm.entrypoints.api_server \
+                   --model $MODEL_NAME \
+                   --tensor-parallel-size $SKYPILOT_NUM_GPUS_PER_NODE \
+                   --tokenizer hf-internal-testing/llama-tokenizer 2>&1 | tee api_server.log &
+
+  echo 'Waiting for vllm api server to start...'
+  while ! `cat api_server.log | grep -q 'Uvicorn running on'`; do sleep 1; done
+
+  echo 'Starting gradio server...'
+  python vllm/examples/gradio_webserver.py
+
+envs:
+  MODEL_NAME: decapoda-research/llama-65b-hf
```

Comment:
I kind of like putting this before `setup`. So when it's used it's defined. Wdyt?

Reply:
Good point! Moved. Thanks!